code/defines/obj/weapon.dm

```diff
@@ -193,16 +193,6 @@
 	w_class = 5.0
 */
 
-/obj/item/weapon/gift
-	name = "gift"
-	desc = "A wrapped item."
-	icon = 'icons/obj/items.dmi'
-	icon_state = "gift3"
-	var/size = 3.0
-	var/obj/item/gift = null
-	item_state = "gift"
-	w_class = 4.0
-

```

Comment:
Remember when dorisdwarf commented on a year old pr removing unused items just to stir up shit?